src/myrtlespeech/configs/deep_speech_2_en.config

```diff
@@ -109,6 +109,7 @@ train_config {
     }
   }
   shuffle_batches_before_every_epoch: true;
+  sortagrad: true;
```

Comment:
The Deep Speech 2 paper describes SortaGrad as:

```
Specifically, in the first training epoch we iterate through minibatches in the training set in increasing order of the length of the longest utterance in the minibatch.  After the first epoch training reverts back to a random order over minibatches.
```

Having both a `shuffle_batches_before_every_epoch` and `sortagrad` option is not consistent with this. i.e. What does `shuffle_batches_before_every_epoch: false; sortagrad: true` mean?

A potential alternative is to have a `shuffle_strategy`-like field:

```
oneof shuffle_strategy {
    Unshuffled unshuffled = 1;
    Random random = 2;
    SortaGrad sorta_grad = 3;
}
```

`Unshuffled` and `Random` are not good names but hopefully this gives an idea.

Thoughts on this?

Reply:
Basically if `sortagrad: true` the `shuffle_batches_before_every_epoch` flag will be ignored in the first epoch, so I agree with the fact that having two separate flags could be a bit confusing for the final user